tests/models/_list_view_model_test.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import todo_app.models.listViewModel as listViewModel
+import todo_app.models.todoItem as todoItem
+
+class MockedCard():
+   def __init__(self, id: str, title: str, status: todoItem.TodoItemStatus):
+      self.id = id
+      self.title = title
+      self.status = status
+
+@pytest.fixture
+def mock_not_started_items():
+   return [
+      MockedCard("test_not_started_0", "not_started_0", todoItem.TodoItemStatus.NOT_STARTED),
+   ]
+
+@pytest.fixture
+def mock_in_progress_items():
+   return [
+      MockedCard("test_in_progress_0", "in_progress_0", todoItem.TodoItemStatus.IN_PROGRESS),
+   ]
+
+@pytest.fixture
+def mock_completed_items():
+   return [
+      MockedCard("test_completed_0", "completed_0", todoItem.TodoItemStatus.COMPLETE),
+   ]
+
+@pytest.fixture
+def mock_list_items(mock_not_started_items, mock_in_progress_items, mock_completed_items):
+    return mock_not_started_items + mock_in_progress_items + mock_completed_items
+
+def test_get_items(mock_list_items):
+   # arrange
+
+   # act
+   view_model = listViewModel.ListViewModel(mock_list_items)
+
+   # assert
+   assert(len(view_model.items) == len(mock_list_items))
```

Comment:
_**Really**_ minor, but I'd quibble over the arrange/act/assert pieces here - I'd argue the action we're doing is to `get_items` based on the test name, so I'd restructure this slightly:

```suggestion
   # arrange
   view_model = listViewModel.ListViewModel(mock_list_items)

   # act
   items = view_model.items

   # assert
   assert(len(items) == len(mock_list_items))
```